Few changes in UI

khairatunhissan opened the pull request.

awhicks reviewed (changes requested):
I changed the branch to staging from main and I think most things were okay with that. Could you run back through and where you've commented code out just go ahead and remove it? It'll be in the git history if we need to go back and get it.

Two things that did not seem to work for me, I never saw the login button appear on the instructions page as I assumed it should, and I think the item links in the search results aren't populating with the new parameters (this view might have also changes slightly from the recent PR and it might be worth merging staging back into your branch before taking a look at that)

Thank you!

khairatunhissan commented:
Hi,
So I have added the login button only on the upload page should it also be
visible at the instruction page. Also I am updating the search page
according to the recent changes.

Thank you for the insights.

On Mon, Apr 14, 2025 at 9:11 PM Alex Hicks ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> I changed the branch to staging from main and I think most things were
> okay with that. Could you run back through and where you've commented code
> out just go ahead and remove it? It'll be in the git history if we need to
> go back and get it.
>
> Two things that did not seem to work for me, I never saw the login button
> appear on the instructions page as I assumed it should, and I think the
> item links in the search results aren't populating with the new parameters
> (this view might have also changes slightly from the recent PR and it might
> be worth merging staging back into your branch before taking a look at that)
>
> Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CSSPLICE/splice-catalog/pull/38#pullrequestreview-2766210894>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANP3NL5P4Y6NR5WMFUN4IJ32ZRMFDAVCNFSM6AAAAAB3BYNJRCVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDONRWGIYTAOBZGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

awhicks commented:
Ah, sorry, I meant the upload page, but below is a screenshot of what I see

![Screenshot 2025-04-14 at 9 39 58 PM](https://github.com/user-attachments/assets/3a58003b-d2c1-41ca-bce7-ef0d9c018915)

awhicks commented:
Okay, I believe I addressed the conflicts correctly but there was one issue. We were trying to refactor the table view out of the catalog and search views into one partial table.ejs that is imported by both. I think that should still work, but can you ensure the table.ejs file has your updates and can be called in both places?
